components/generic/SimilarItems.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section class="row py-4 px-2 bg-white">
+    <b-col>
+      <b-card-group
+        class="card-deck-4-cols similar-items"
+        deck
+      >
+        <ContentCard
+          v-for="(item, index) in items"
+          :key="index"
+          :title="item.name"
```

Comment:
As we are passing `title` down, let's make it `title` on the items too.

Reply:
Sorry, not sure what you mean here

Reply:
Change the property name on items to be `title`, i.e. `item.title`